src/util.h

```diff
@@ -88,7 +100,9 @@ void CreatePidFile(const fs::path &path, pid_t pid);
 #ifdef WIN32
 fs::path GetSpecialFolderPath(int nFolder, bool fCreate = true);
 #endif
+#if !(TARGET_OS_IPHONE || TARGET_IPHONE_SIMULATOR)
```

Comment:
Check if `TARGET_OS_IPHONE` is defined before using. Please fix throughout: applies to all reading of `TARGET_*` in `util.h`, `init.cpp`, `validation.cpp`, `util.cpp`, `wallet/init.cpp` and `wallet/wallet.cpp`.

Reply:
I replaced them all with `#if defined(HAVE_STD__SYSTEM) || defined(WIN32)
`, which is now set in configure.ac.